packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -46,28 +46,43 @@ const sanitizePathSegment = name => {
  * @typedef {import('../reserve/assetReserve.js').GovernedAssetReserveFacetAccess} GovernedAssetReserveFacetAccess
  */
 
+/**
+ * redundant with utils.d.ts
+ *
+ * @typedef {object} AdminFacet
+ * @property {() => PromiseLike} getVatShutdownPromise
+ * @property {(contractBundleId: string, newPrivateArgs: any) => void} upgradeContract
+ * @property {(newPrivateArgs:any) => void} restartContract
+ */
+
 /**
  * @typedef { WellKnownSpaces & ChainBootstrapSpace & EconomyBootstrapSpace
  * } EconomyBootstrapPowers
  * @typedef {PromiseSpaceOf<{
  *   ammInstanceWithoutReserve: Instance,
  *   ammCreatorFacet: XYKAMMCreatorFacet,
  *   ammGovernorCreatorFacet: GovernedContractFacetAccess<XYKAMMPublicFacet,XYKAMMCreatorFacet>,
+ *   ammGovernorAdminFacet: AdminFacet,
```

Comment:
hm. I suppose this works, but what I had in mind was to capture the whole result of `E(zoe).startInstance(ammGovernorInstallation)` in one field, replacing `ammGovernorCreatorFacet`.

Reply:
Pervasively done.